nss-day-cohort-41/react-nutshell-brazil-nuts
# Bd events 2

# Description : Added Content to Events components
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

## Type of change Added POST methods
Please delete options that are not relevant.


# Testing Instructions
To test this you can log in as a user, and in the events section use the form to create an event. It should then create a new card for the event list. The event list will only populate with events that are the active users. Delete button deletes the event from the site and the database.

# Checklist:
- [ x] My code follows the style guidelines of this project
- [ x] I have performed a self-review of my own code
- [ x] I have added test instructions that prove my fix is effective or that my feature works
